doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/implementation/digital-receipt-implementation
+title: 'Digital receipt implementation'
+---
+
+# Digital receipt transmission
+
+>[!IMPORTANT]
+>Before start implementing, please read the getting started section first to make sure you are choosing the best suiting transmitting method for your Point of Sales software. 
+
+fiskaltrust provides two distinct approaches for transmitting receipt requests to the fiskaltrust.Middleware. The first approach is the POS API Helper, which is primarily recommended for testing/sandbox environments and the InStore App. Configuring the POS API Helper within the fiskaltrust.Portal requires minimal implementation effort in your Point of Sale software.
+
+However, it's important to highlight that the POS API Helper does not log the delivery statuses of the digital receipt, as mentioned in the section Evaluation of document retrievals for financial administration (Finanzverwaltung). The absence of these logs prevents a tax auditor from reviewing the statuses of printing, acceptance, and submission in the event of an audit. This could result in non-compliance, particularly in Austria, due to the lack of logged records for "Belegausgabepflicht" and "Belegannahmepflicht," rendering verification impossible.
```

Comment:
Better to use english descriptions for Belegausgabepflicht and Belegannahmepflicht as well

Reply:
Also in the paragraphs below